### ISSUE TITLE
Can't save settings

### ISSUE THREAD

**Issue by donnib** [stuartpittaway/diyBMSv4Code#98]
I have seen this issue few times.

![error](https://user-images.githubusercontent.com/252702/107047772-7ccf1b80-67c8-11eb-98ba-c24beff7556f.gif)

**Comment by stuartpittaway** [comment]
This is by design.  It prevents cross site scripting attacks, this shouldn't be a problem unless the controller has been rebooted and you didn't refresh the webpage.

**Comment by donnib** [comment]
@stuartpittaway the last thing might have happened yes. Maybe the wording could be, Failed to save settings, please make a refresh or something like it ?

**Comment by jbuszkie** [comment]
I just hit this too...  Refreshing the browser fixed it.  I agree with @donnib.  Adding something to the warning might help.

**Comment by stuartpittaway** [comment]
This occurs when you reboot the controller, but don't refresh the web page

**Comment by jbuszkie** [comment]
Yeah that's what happened.  But maybe you can change the "Failed to save settings" to include something like "Failed to save settings. Try refreshing the browser"  Or something like that...

Thanks!
